routing/routing.xml

```diff
@@ -953,7 +1055,7 @@
 
 		</point>
 
-		<way attribute="penalty_transition">
+		<!-- <way attribute="penalty_transition">
```

Comment:
It was added because of frequently routing engine attempts to avoid main highway which starts with left turn. Left turn on main highways is very dangerous for bicycle and in some countries it is not allowed at all. So I disagree with this change.